features/context/MfaContext.php

```diff
@@ -416,22 +423,15 @@ public function thereShouldBeAWayToContinueToMyIntendedDestination()
      */
     public function iClickTheRemindMeLaterButton()
     {
-        $page = $this->session->getPage();
-        $continueButton = $this->getContinueButton($page);
-        Assert::assertNotNull($continueButton, 'Failed to find the continue button');
-        $continueButton->click();
-        $this->submitSecondarySspFormIfPresent($page);
+        $this->submitFormByClickingButtonNamed('continue');
```

Comment:
I really like this method!

Reply:
Thanks! :-) There had been enough duplication there that I realized there was a better way.